test/remoting.integration.js

```diff
@@ -6,6 +6,12 @@ var app = require(path.join(SIMPLE_APP, 'server/server.js'));
 var assert = require('assert');
 
 describe('remoting - integration', function() {
+  before(function(done) {
+    if (app.booting) {
+      return app.once('booted', done);
+    }
+    done();
+  });
```

Comment:
Can we move this block into `lt.beforeEach.withApp`, so that we don't have to duplicate it?